index.bs

```diff
@@ -1454,7 +1454,7 @@ authenticator over a set of contextual bindings. Each attestation statement form
     [=hash of the serialized client data=], and returns either:
     - An error indicating that the attestation is invalid, or
     - The attestation type, and the trust path of the attestation. This trust path is either empty (in case of
-        self-attestation), a DAA root key (in the case of Direct Anonymous Attestation), or a set of X.509 certificates.
+        self-attestation), an identifier of a [=DAA Issuer public key=] (in the case of Direct Anonymous Attestation), or a set of X.509 certificates.
```

Comment:
Yes.  Someone changed it from ECDAA to DAA some time ago.  Will revert back to ECDAA.